lib/avrora/storage/registry.ex

```diff
@@ -93,10 +93,26 @@ defmodule Avrora.Storage.Registry do
   @spec configured?() :: true | false
   def configured?, do: !is_nil(registry_url())
 
+  defp csr_references_lookup_function(response) do
+    case Map.fetch(response, "references") do
+      {:ok, csr_references} ->
+        references = Enum.reduce(csr_references, %{}, fn(cr, map) ->
+          {:ok, schema_map} = get(cr["subject"])
+          Map.put(map, schema_map.full_name, schema_map.json)
+        end )
+
+        fn(r) ->
+          json_schema = Map.get(references, r)
+          {:ok, json_schema}
+        end
```

Comment:
```suggestion
        {:ok, &Map.fetch(references, &1)}
```

Reply:
@Strech thank you for your review, I'm a newbie in Elixir so any correction is very important for me.
I think I've applied all your corrections in last commit. Please take a look and if I missed something please let me know. Thanks.